src/main/kotlin/com/github/pool_party/pull_party_bot/commands/PingCommandHandlers.kt

```diff
@@ -174,13 +152,13 @@ private fun Bot.handleAdminsParty(msg: Message): String? {
 /**
  * Delete given parties from DataBase.
  */
-fun Bot.handleDelete(msg: Message, args: String?) {
+val delete = newCommand("delete", "forget the parties as they have never happened", HELP_DELETE) { msg, args ->
```

Comment:
Supposed to be admin command

Reply:
Creator cannot delete it's own party, is that ok?

Reply:
We don't store the information about creator, so how would yuo do that?